src/simd_128_u16.h

```diff
@@ -174,7 +164,7 @@ inline void mul_coef_to_buf(
 }
 
 inline void
-add_two_bufs(aint16* src, aint16* dest, size_t len, aint16 card = F3)
+add_two_bufs_u16(aint16* src, aint16* dest, size_t len, aint16 card = F3)
```

Comment:
I think these `_u16` suffix are useless and can be removed (same comment for `mul_coef_to_buf`, `sub_two_bufs` and `mul_two_bufs` for both `_u16` and `_u32` suffix)

Reply:
it's done